platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -352,7 +352,7 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
             if (path == podApiFile) {
                 val assetPath = "container/pod.js"
                 logger.debug("$podApiFile requested - returning $assetPath")
-                if (featureFile.getEntry(podApiFile) != null) {
+                if (File(featureFilePath.plus("/$podApiFile")).exists()) {
```

Comment:
Or maybe check this before the call and make the call with a `File` argument?

Reply:
This function needs to work with the path, like appending to it.